packages/react-native-editor/package.json

```diff
@@ -123,10 +123,10 @@
 		"clean:runtime": "npm run clean:haste; npm run clean:react; npm run clean:metro; npm run clean:jest; npm run clean:watchman; npm run clean:babel-cache",
 		"clean:build-artifacts": "rm -rf ./ios/build && rm -rf ./ios/Pods",
 		"clean:aztec": "cd ../react-native-aztec && npm run clean",
-		"clean:haste": "rm -rf /tmp/haste-map-react-native-packager-*",
+		"clean:haste": "rm -rf $TMPDIR/haste-map-metro-*",
 		"clean:install": "npm run clean; npm install",
 		"clean:jest": "jest --clearCache --config ../../test/native/jest.config.js; rm -rf $TMPDIR/jest_*",
-		"clean:metro": "rm -rf $TMPDIR/metro-cache-*;",
+		"clean:metro": "rm -rf $TMPDIR/metro-cache/",
 		"clean:react": "rm -rf $TMPDIR/react-*",
```

Comment:
I'm not seeing any `$TMPDIR/react-*` files. Perhaps we don't need the `clean:react` command anymore?

Reply:
I don't have any `$TMPDIR/react-*` either, probably ok to remove the command.

Reply:
👍  Updated the PR to remove `clean:react` as well. I'll re-request your review in case you have any additional thoughts.